src/main/java/org/codehaus/mojo/versions/SetPropertyMojo.java

```diff
@@ -63,6 +66,15 @@
     @Parameter( property = "autoLinkItems", defaultValue = "true" )
     private boolean autoLinkItems;
 
+    /**
+     * A property file name containing:
```

Comment:
This is not rendered properly. Please run `mvn site` and check the site generated under `target/site`.

Reply:
Fixed!